### PR TITLE
add more information to the printed error code

### PR DESCRIPTION
**Pull request by barcode**
now you get more information than a number.  
the description was taken from the user guide

### CODE REVIEW

**Comment by gergondet** [comment]
LGTM :+1: 

Seems like a recent change in eigen-qld API broke the QLD backend in Tasks. I fixed it in master and took the liberty to rebase your PR. I'll merge once the tests pass.

**Comment by haudren** [comment]
This should not be added to Tasks: we already have an "inform()" function that does just that in `eigen-lssol`.

**Comment by gergondet** [comment]
Woops. Did not know that. Then it's still probably better to call inform than leave it in the current state.

**Comment by haudren** [comment]
Even if that function did not exist, I'd rather have most of the solver-specific code inside their `eigen-*` packages, so that it can be used by external applications.

**Comment by barcode** [comment]
now the inform function is used (i did not know it existed).
this branch now needs a small patch to `eigen-lssol` (there is a merge request).

File: src/LSSOLQPSolver.cpp [hunk 1]
```diff
@@ -161,7 +161,10 @@ std::ostream& LSSOLQPSolver::errorMsg(
 {
 	const int nrVars = int(Q_.rows());
 
-	out << "lssol output: " << lssol_.fail() << std::endl;
+	out << "lssol output (" << lssol_.fail() << "): ";
+	lssol_.inform(out);
+	out << std::endl;
```
Comment (gergondet) [inline src/LSSOLQPSolver.cpp]: I think this should be after the output code to have a nice format.

**Pull request merged**